### ISSUE TITLE
Lockscreen can only use numeric passwords

### ISSUE THREAD

**Issue by ollieparanoid** [postmarketOS/pmbootstrap#1433]
#1423 fixed scaling on high DPI devices, but also enabled a feature that presents a numeric password input to the user. This makes it impossible to type in non-numeric passwords. As workaround, assign your user in the postmarketOS installation a numeric password.

@bhush9 wrote in this context:
> I ummm.. let's see if I can get this fixed Saturday

**Comment by montvid** [comment]
Yeah, plasma tells me to enter some numbers on lockscreen. Just disable it if no one can fix it.
